.travis.yml

```
@@ -99,7 +102,7 @@ before_script:
 - cd ..
 
 script:
-- travis_wait 50 coverage run --source=fonduer -m pytest tests
+- travis_wait 60 coverage run --source=fonduer -m pytest tests
```

```suggestion
- travis_wait 50 coverage run --source=fonduer -m pytest tests
```

This should remain 50, as 50 min is the upper-limit on a Travis build anyways.